app/src/components/Api.tsx

```diff
@@ -0,0 +1,15 @@
+export const PRODUCTION =
```

Comment:
This should be a `.ts` file, not a `.tsx` file since it doesn't contain any components.